seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/rest/RestService.java

```diff
@@ -0,0 +1,674 @@
+                        + "failed, status code is "
+                        + statusCode
+                        + ".";
+        throw new DorisConnectorException(DorisConnectorErrorCode.REST_SERVICE_FAILED, errMsg, ex);
+    }
+
+    private static String getConnectionPost(
+            HttpRequestBase request, String user, String passwd, Logger logger) throws IOException {
+        URL url = new URL(request.getURI().toString());
+        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+        conn.setInstanceFollowRedirects(false);
+        conn.setRequestMethod(request.getMethod());
+        String authEncoding =
+                Base64.getEncoder()
+                        .encodeToString(
+                                String.format("%s:%s", user, passwd)
+                                        .getBytes(StandardCharsets.UTF_8));
+        conn.setRequestProperty("Authorization", "Basic " + authEncoding);
+        InputStream content = ((HttpPost) request).getEntity().getContent();
+        String res = IOUtils.toString(content);
+        conn.setDoOutput(true);
+        conn.setDoInput(true);
+        PrintWriter out = new PrintWriter(conn.getOutputStream());
+        // send request params
+        out.print(res);
+        // flush
+        out.flush();
+        // read response
+        return parseResponse(conn, logger);
+    }
+
+    private static String getConnectionGet(
+            String request, String user, String passwd, Logger logger) throws IOException {
+        URL realUrl = new URL(request);
+        // open connection
+        HttpURLConnection connection = (HttpURLConnection) realUrl.openConnection();
+        String authEncoding =
+                Base64.getEncoder()
+                        .encodeToString(
+                                String.format("%s:%s", user, passwd)
+                                        .getBytes(StandardCharsets.UTF_8));
+        connection.setRequestProperty("Authorization", "Basic " + authEncoding);
+
+        connection.connect();
+        return parseResponse(connection, logger);
+    }
+
+    private static String parseResponse(HttpURLConnection connection, Logger logger)
+            throws IOException {
+        if (connection.getResponseCode() != HttpStatus.SC_OK) {
+            logger.warn(
+                    "Failed to get response from Doris  {}, http code is {}",
+                    connection.getURL(),
+                    connection.getResponseCode());
+            throw new IOException("Failed to get response from Doris");
+        }
+        StringBuilder result = new StringBuilder();
+        BufferedReader in =
+                new BufferedReader(
+                        new InputStreamReader(connection.getInputStream(), StandardCharsets.UTF_8));
+        String line;
+        while ((line = in.readLine()) != null) {
+            result.append(line);
+        }
+        if (in != null) {
+            in.close();
+        }
```

Comment:
it's better to used `try catch reource`, and L226 is not null forever

Reply:
done